documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+|                                           | **Use Case Example 2 - Network Impairments**<br>Issues in the operator's network can lead to degradation in latency, jitter, packet loss, and bitrate, reflecting lower QbD scores and poor video quality.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|                                           | **Sample Recommendations**<br>- create a HomeDeviceQoD session for WMM enablement<br>- application configuration updates<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| Technical viability                       | This API is for application developers to consume. It assumes the network operator exposing this API has a backend to receive KPIs, analyze them, and produce a QbD score. Based on thresholds for the app, the network operator can provide proactive recommendations. The application could also use the Camara NetworkAccessManagement for remote control of a [`Device`](https://github.com/camaraproject/NetworkAccessManagement/blob/1c804a5159f67370f005dbf0cc82e5c7e72725f0/code/API_definitions/network_access_management.yaml#L34) at a [`Service Site`](https://github.com/camaraproject/NetworkAccessManagement/blob/1c804a5159f67370f005dbf0cc82e5c7e72725f0/code/API_definitions/network_access_management.yaml#L40) if necessary. |
```

Comment:
Revisiting this... is consume the right word? I'd suggest "This API is for application developers interact with the network".

Reply:
How about "Based on "minimum quality thresholds for the application category"?

Reply:
CAMARA should be all caps for consistency.

Reply:
good feedback. I've incorporated - thx @shafayetkhan